modular_zubbers/code/datums/weather/weather_types/acid_rain.dm

```diff
@@ -0,0 +1,154 @@
+/datum/weather/acid_rain
+	name = "acid rainstorm"
+	desc = "The planet's thunderstorms are by nature acidic, and will incinerate anyone standing beneath them without protection."
+	probability = 90
+
+	telegraph_message = span_userdanger("Thunder rumbles far above. You hear droplets drumming against the canopy. Seek shelter.")
+	telegraph_duration = 400
+	telegraph_sound = "modular_zubbers/sound/ambience/acidrain_start.ogg"
```

Comment:
not a valid sound file path, you must use 'soundfile' instead of "soundfile", the ' is important here